engines/interpolantmc.h

```diff
@@ -30,11 +30,10 @@ class InterpolantMC : public Prover
 {
  public:
   // IMPORTANT: assume the property was built using the interpolating solver
-  InterpolantMC(const Property & p, smt::SmtSolver & slv, smt::SmtSolver & itp);
-  InterpolantMC(const PonoOptions & opt,
-                const Property & p,
-                smt::SmtSolver & slv,
-                smt::SmtSolver & itp);
+  InterpolantMC(const Property & p, smt::SolverEnum se);
+  InterpolantMC(const Property & p, smt::SmtSolver slv, smt::SmtSolver itp);
+  InterpolantMC(const PonoOptions & opt, const Property & p, smt::SolverEnum se);
+  InterpolantMC(const PonoOptions & opt, const Property & p, smt::SmtSolver slv, smt::SmtSolver itp);
```

Comment:
const ref to SmtSolver

Reply:
Thanks, addressed with 0dbc624d1d5d9ab57c7977b758211620fdff5ecd